javascript/builtins/intl/Intl.json

```diff
@@ -104,6 +104,58 @@
               "deprecated": false
             }
           }
+        },
+        "@@toStringTag": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Intl/@@toStringTag",
+            "spec_url": "https://tc39.es/ecma402/#sec-Intl-toStringTag",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": {
+                "version_added": false
+              },
+              "edge": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": "83"
+              },
+              "firefox_android": {
+                "version_added": "83"
+              },
+              "ie": {
+                "version_added": false
+              },
+              "nodejs": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": false
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": false
```

Comment:
Safari 14 supports it
https://trac.webkit.org/changeset/262568/webkit

Reply:
Ah, no. `Intl[@@toStringTag]` is https://trac.webkit.org/changeset/266015/webkit, and it is not included in Safari 14 yet.